kornia/geometry/camera/stereo.py

```diff
@@ -0,0 +1,287 @@
+import torch
+
+
+class StereoException(Exception):
+    def __init__(self, msg, *args, **kwargs):
+        """
+        Custom exception for the :module:`~kornia.geometry.camera.stereo` module. Adds a general helper module
+        redirecting the user to the proper documentation site.
+        Args:
+            msg: Custom message to add to the general message.
+            *args:
+            **kwargs:
+        """
+        doc_help = (
+            "\n Please check documents here: "
+            "https://kornia.readthedocs.io/en/latest/geometry.camera.stereo.html"
+            "for further information and examples."
+        )
+        final_msg = msg + doc_help
+        super().__init__(final_msg, *args, **kwargs)  # type: ignore
```

Comment:
```suggestion
        super().__init__(final_msg, *args, **kwargs)
```
why type ignore?

Reply:
I think my editor complained, but it can be removed.